domains/Discussions/Tests/Feature/QuestionsViewTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Domains\Discussions\Tests\Feature;
+
+use Domains\Accounts\Database\Factories\UserFactory;
+use Domains\Accounts\Models\User;
+use Domains\Discussions\Database\Factories\QuestionFactory;
+use Domains\Discussions\Models\Question;
+use Illuminate\Http\Response;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class QuestionsViewTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    private User $user;
+    private Question $question;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->user     = UserFactory::new()->create();
+        $this->question = QuestionFactory::new(['author_id' => $this->user->id])->create();
+    }
+
+    /** @test */
+    public function guest_can_see_a_question(): void
```

Comment:
minor: to keep consistency, can we name the test functions` it...`, like
`it_allows_guest_to_see_a_question`